installer.js

```diff
@@ -1,29 +1,29 @@
-const { promisify } = require('util');
-
-const fs = require('fs');
-const statAsync = promisify(fs.stat);
-const existsAsync = promisify(fs.exists);
-const mkdirAsync = promisify(fs.mkdir);
-const readdirAsync = promisify(fs.readdir);
-const copyFileAsync = promisify(fs.copyFile);
+const fs = require('fs').promises;
+const { F_OK } = require('fs').constants;
 
 const path = require('path');
 const makensis = require('./makensis');
 
 const isDirectoryAsync = async (item) => {
-    const stats = await statAsync(item);
+    const stats = await fs.stat(item);
     
     return stats.isDirectory();
 };
 
+const fileExists = async (item) => {
+    return await fs.access(item, F_OK)
+        .then(() => true)
+        .catch(() => false);
```

Comment:
Any reason not to use the following syntax?
```suggestion
    try {
        await fs.access(item, F_OK);
        return true;
    }
    catch {
        return false;
    }
```

Reply:
Alternately - Would it make sense to keep what's there, and then drop `async` and `await` from the declaration since it is handling the promise with `then` and `catch`?

Reply:
I actually prefer async/await myself, but on my first try the test failed. I then found this example online. However, the mistake must've been elsewhere, because now it's working! :)